### DISCUSSION TITLE
Internal integrations

### DISCUSSION THREAD

**Discussion by 2e12** [Budibase/budibase#1230] [Help] [answered]
Hey! 

Is it possible to create and run integrations yourself without having to publish them publicly? Are there any best practices?

Thanks already for your help! 😄

**Comment by shogunpurple** [comment, marked as answer]
Hi @2e12 - thanks for taking the time to raise a discussion. You cannot currently create and run integrations yourself without contributing them into the main repo. This is on our roadmap and we will have a full development experience that allows users to do this over the coming months.

**Reply by 2e12** [reply]
@shogunpurple Thank you for the feedback. We will be curious to see what changes are still to come. 😃
